cli/tests/pcluster/config/test_cluster_config.py

```diff
@@ -327,6 +327,7 @@ def base_cluster_config(self):
         return BaseClusterConfig(
             cluster_name="clustername",
             image=Image("alinux2"),
+            disable_sudo_access_default_user=True,
```

Comment:
Do we have to use this?
I believe we only need "required" parameters?

Reply:
Not really I think we can skip this